### PR TITLE
feat: handles monika authorization

### PR DESCRIPTION
**Pull request by kevinhermawan**
# Symon Pull Request (PR)

### what issue does this pr address

1. implementing issue no #153 which adds monika authorization using api key

### how **DID** you test

1. npx prisma migrate dev --preview-feature
2. npx prisma generate
3. npm start
4. npm test:server
5. hit `monika/*` routes using header `x-api-key`

### CODE REVIEW

File: src/server/prisma/migrations/20210423084418_/migration.sql [hunk 1]
```diff
@@ -1,8 +0,0 @@
-/*
```
Comment (haricnugraha) [inline src/server/prisma/migrations/20210423084418_/migration.sql:1]: You don't need to remove migration file every time you change database structure, migration file is like version control for database. If you remove migration file, you lose database changes history.

Comment (kevinhermawan) [reply src/server/prisma/migrations/20210423084418_/migration.sql]: Uh oh, ok thanks!

File: src/server/prisma/migrations/20210427083725_/migration.sql [hunk 2]
```diff
@@ -0,0 +1,82 @@
+-- CreateTable
+CREATE TABLE "user" (
```
Comment (raosan) [inline src/server/prisma/migrations/20210427083725_/migration.sql:2]: Same as @haricnugraha 's comment. you don't need to create new table that has been created before. just add the additional/changes.

Comment (kevinhermawan) [reply src/server/prisma/migrations/20210427083725_/migration.sql:2]: It generated by prisma

**Review by raosan — APPROVED** [review]

**Review by armedi — APPROVED** [review]

**Pull request merged**